src/App.js

```diff
@@ -1,28 +1,31 @@
+import React, { useState } from "react";
+import data from './data.json';
+//components
+import Header from './header';
+import ToDoList from "./ToDoList";
+
 import './App.css';
-import Header from "./components/Header";
-import React from "react";
 
 function App() {
+    const [ toDoList, setToDoList] = useState(data);
+
+    const handleToggle = (id) => {
+        let mapped = toDoList.map(task => {
+            return task.id == id ? {...task, complete: !task.complete} : {...task};
```

Comment:
Ternary conditionals are good for short code, but they can lead to hard to understand code too :) 
Could you rewrite the row 14 in the old school way with classing if else statement?